libraries/Microsoft.Bot.Configuration/BotConfigurationEndpointServiceCredentialProvider.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Bot.Configuration;
+using Microsoft.Bot.Connector.Authentication;
+
+namespace Microsoft.BotBuilderSamples
```

Comment:
Probably wrong namespace..

Reply:
Yikes, yes, completely. I prototyped under samples and when I moved the file in here I didn't catch that.

Thank you, I will make sure to move it, but I guess what namespace it ends up in will go hand in hand with the decision about what assembly it will go in. 🤔